<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addlist</class>
 <widget class="QDialog" name="addlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="surname1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="experience1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>41</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>51</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Subject</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Experience</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>